### PR TITLE
feat(no-ref): add reusable card-content

### PR DESCRIPTION
**Pull request by andriikamaldinov1**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/JsDaddy/ngx-mask/blob/develop/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/app/options/options.component.html [hunk 1]
```diff
@@ -43,54 +43,46 @@
...
-                </div>
-            </div>
-            <div class="card-text column pd-15">
-                <span class="sub-text row full-white title ls-25">Mask:</span>
-                <span class="green-view ai-start span">{{ ex._mask }}</span>
+                <ngx-mask-demo-card-content
+                    [value]="ex.control.form.value | isEmpty"
+                    [title]="'Form Control:'"
+                    [color]="'yellow-view'">
+                </ngx-mask-demo-card-content>
```
Comment (GlebChiz) [inline src/app/options/options.component.html:50]: [title]="'Form Control:'" [color]="'yellow-view'" 
-> 
title="Form Control:" color="yellow-view"

File: src/app/options/options.component.html [hunk 2]
```diff
@@ -43,54 +43,46 @@
...
+                <ngx-mask-demo-card-content
+                    [value]="ex.control.model | isEmpty"
+                    [title]="'Ng Model:'"
+                    [color]="'yellow-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:54]: [title]="'Form Control:'" [color]="'yellow-view'" 
-> 
title="Form Control:" color="yellow-view"

File: src/app/options/options.component.html [hunk 3]
```diff
@@ -43,54 +43,46 @@
...
+                </ngx-mask-demo-card-content>
             </div>
+            <ngx-mask-demo-card-content [value]="ex._mask" [title]="'Mask:'" [color]="'green-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:57]: [title]="'Form Control:'" [color]="'yellow-view'" 
-> 
title="Form Control:" color="yellow-view"

File: src/app/options/options.component.html [hunk 4]
```diff
@@ -43,54 +43,46 @@
...
-            }}</span>
-        </div>
-        <div class="card-text column pd-15">
-            <span class="sub-text full-white title ls-25">mask</span>
-            <span class="green-view ai-start span">{{ ex._mask }}</span>
-        </div>
+        <ngx-mask-demo-card-content
+            [value]="ex.control.form.errors | json"
+            [title]="'Form Control:'"
+            [color]="'yellow-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:65]: [title]="'Form Control:'" [color]="'yellow-view'" 
-> 
title="Form Control:" color="yellow-view"

File: src/app/options/options.component.html [hunk 5]
```diff
@@ -43,54 +43,46 @@
...
+        </ngx-mask-demo-card-content>
+        <ngx-mask-demo-card-content
+            [value]="ex.control.form.errors | json"
+            [title]="'Mask:'"
+            [color]="'green-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:70]: The same

File: src/app/options/options.component.html [hunk 6]
```diff
@@ -43,54 +43,46 @@
...
-        <div class="card-text column pd-15">
-            <span class="sub-text full-white title ls-25">Value:</span>
-            <span class="green-view ai-start span">123456789</span>
-        </div>
+        <ngx-mask-demo-card-content [value]="phone | mask : '(000) 000-0000'">
+        </ngx-mask-demo-card-content>
+        <ngx-mask-demo-card-content
+            [value]="'(000) 000-0000'"
+            [title]="'Mask:'"
+            [color]="'green-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:82]: The same

File: src/app/options/options.component.html [hunk 7]
```diff
@@ -43,54 +43,46 @@
...
+        </ngx-mask-demo-card-content>
+        <ngx-mask-demo-card-content [value]="'123456789'" [title]="'Value:'" [color]="'green-view'">
```
Comment (GlebChiz) [inline src/app/options/options.component.html:84]: The same

File: src/app/shared/card-content/card-content.component.html [hunk 8]
```diff
@@ -0,0 +1,4 @@
+<div class="card-text column pd-15">
+    <span class="sub-text full-white title ls-25">{{ title }}</span>
+    <span class="ai-start span yellow" [ngClass]="color">{{ value }}</span>
```
Comment (GlebChiz) [inline src/app/shared/card-content/card-content.component.html:3]: May be better this way?
class="ai-start span yellow" [ngClass]="color" -> [ngClass]="['ai-start span yellow', color]"

File: src/app/shared/card-content/card-content.component.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
+@import 'src/styles';
+.card-text {
+    .sub-text {
+        margin-bottom: 5px;
+    }
+    .green-view {
+        color: #1ab77e;
```
Comment (GlebChiz) [inline src/app/shared/card-content/card-content.component.scss:7]: create variable for this color

File: src/app/shared/card-content/card-content.component.ts [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+    selector: 'ngx-mask-demo-card-content',
+    templateUrl: './card-content.component.html',
+    styleUrls: ['./card-content.component.scss'],
+    standalone: true,
+    imports: [NgClass],
+})
+export class CardContentComponent {
+    @Input() public color!: string;
+    @Input() public title!: string;
+    @Input() public value!: string;
```
Comment (GlebChiz) [inline src/app/shared/card-content/card-content.component.ts:14]: If this component can not be without any field, add this field to selector
For example: title and value are required, then selector will be 'ngx-mask-demo-card-content[title][value]'

File: src/app/shared/card-content/card-content.component.scss (hunk 9 above)
Comment (andriikamaldinov1) [reply src/app/shared/card-content/card-content.component.scss]: did it

**Review by GlebChiz — APPROVED** [review]

**Pull request merged**